packages/circle-demo-webapp/app/tailwind.css

```diff
@@ -19,7 +19,7 @@
     --card-foreground: 240 10% 3.9%;
     --popover: 0 0% 100%;
     --popover-foreground: 240 10% 3.9%;
-    --primary: 240 5.9% 10%;
+    --primary: 255 82% 64%;
```

Comment:
change primary color to Circle's Jelly 400 as described here https://brand.circle.com/document/198311#/circle/color-1